mapusaurus/respondents/views.py

```diff
@@ -76,6 +78,8 @@ class Meta:
 @api_view(['GET'])
 def search_results(request):
     query_str = escape(request.GET.get('q', '')).strip()
+    year = escape(request.GET.get('year', '')).strip()
+    request.session['year'] = year
```

Comment:
Tests are only failing here because of a bug in the mock `request` parameter that gets passed into `search_results` in `respondents/tests.py`. Specifically the `session` middleware is missing from the `request` produced by `RequestFactory` in that test . [Here's](http://www.mellowmorning.com/2011/04/18/mock-django-request-for-testing/) a general fix.